turbopack/crates/turbo-tasks-backend/src/lmdb_backing_storage.rs

```diff
@@ -418,10 +410,6 @@ impl BackingStorage for LmdbBackingStorage {
             tx.commit()
                 .with_context(|| anyhow!("Unable to commit operations"))?;
         }
-        println!(
-            "Persisted {op_count} db entries after {:?}",
-            start.elapsed()
-        );
```

Comment:
You could add the `op_count` in the parent `save_snapshot` span using `Span::current` and `Span::record`: https://docs.rs/tracing/latest/tracing/struct.Span.html#method.record